### ISSUE TITLE
[OpenJO] Elevator Stuck (Soft Lock)

### ISSUE THREAD

**Issue by SilentMRG** [JACoders/OpenJK#1157]
**Operating system and version:**
Windows 7 Ultimate 64 bits.
I'm using the latest "stable" version of OpenJO.

**Is this for single player or multiplayer?**
single player

**Description of the bug (and if possible, steps to reproduce the bug):**
After releasing Lando and going to meet Reelo Baruk and after all the scene and fight... I went back to talk to Lando and we left to go after his ship "Lady Luck". To finish the level you need to take an elevator, and that's exactly where the bug occurs: The elevator simply doesn't work and pressing the panel doesn't work either, "noclip" doesn't help and I'm stuck in this part.

**What did you expect to happen instead?**
The elevator needs to take Kyle and Lando up.

**Comment by SilentMRG** [comment]
Well, I found a solution: I activated "noclip" and went up as much as I could until "distort the textures", then I deactivated "noclip" and in the Kyle's falling down the game started to the next level. A rudimentary but useful solution, hehehe.

**Comment by SilentMRG** [comment]
Well, well, I found other elevators that don't work, it seems that all the elevators are completely broken... Unfortunately, the "noclip" doesn't work on some elevators and now I'm stuck in Cloud City on that elevator that must be triggered after using Mind Trick on the guy on the panel. Honestly, I'm going to give up on this "OpenJO" source port and go back to playing the vanilla version.

**Comment by ensiform** [comment]
Have you tried with the 32 bit version of openjo or only the 64 bit version ?

**Comment by SilentMRG** [comment]
Well, I downloaded OpenJO from here: https://jk2t.ddns.net/#openjo
I believe it is the 32-bit version.

**Comment by SilentMRG** [comment]
Hey...

Me again. XD I have a question: How do I run Jedi Outcast in OpenJK? Moving the ".pk3" files to the "base" directory I know is not the case, because the Jedi Academy files are there, so when I move the JO files to the "OpenJK" directory I get a menu without letters. If I do the opposite: Move the JK files to the "OpenJK" directory I get an error, and the JO files in the "base" directory do nothing.

I hope I was clear. English is not my native language, hehe.

Thank you for your attention!

**Comment by SilentMRG** [comment]
Here we go...

Well, well, I just tested the OpenJO version from Tuesday, ‎8‎ February, ‎2022, ‏‎17:08:42 downloaded from here: https://jkhub.org/files/file/4008 -openjo-for-jedi-outcast/ And I no longer have soft locks on the elevators and everything works like a charm (as far as I could see)! I used a save game from another version (I don't know if save games are compatible between versions) to do the tests, just so you know. Ultimately, it is up to you to close or not close this topic. ;)

Cheers!
